play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala

```diff
@@ -16,11 +16,20 @@ import play.shaded.ahc.org.asynchttpclient.HttpResponseBodyPart
  * Note that this is only usable with a stream call, i.e.
  *
  * {{{
- * class MyClass extends StreamedBodyReadable {
- *   ws.url("http://example.com").stream().map { response =>
- *      val source = response.body[Source[ByteString, NotUsed]]
- *      ...
- *   }
+ * import scala.concurrent.ExecutionContext
+ *
+ * import akka.util.ByteString
+ * import akka.stream.scaladsl.Source
+ *
+ * import play.api.libs.ws.DefaultBodyReadables._
+ * import play.api.libs.ws.ahc.StandaloneAhcWSClient
+ *
+ * class MyClass(ws: StandaloneAhcWSClient) {
```

Comment:
Good call, doesn't need the import

Reply:
To make the code example standalone, the import is required

Reply:
yes, agreed